src/components/metadata/metadata.test.js

```diff
@@ -30,42 +31,41 @@ describe('MetaData', () => {
   const title = (wrapper) => wrapper.find('.pipeline-metadata__title');
   const rowIcon = (row) => row.find('svg.pipeline-metadata__icon');
   const rowValue = (row) => row.find('.pipeline-metadata__value');
+  const rowObject = (row) => row.find('.pipeline-metadata__object');
   const rowByLabel = (wrapper, label) =>
     // Using attribute since traversal by sibling not supported
     wrapper.find(`.pipeline-metadata__row[data-label="${label}"]`);
 
   describe('All nodes', () => {
-    it('limits parameters to 10 values and expands when button clicked', () => {
-      // Get metadata for a sample
+    it('limits Metadata list to 10 values and expands when button clicked', () => {
       const metadata = getClickedNodeMetaData(
         prepareState({
           data: animals,
           afterLayoutActions: [() => toggleNodeClicked(salmonTaskNodeId)],
         })
       );
-      // Add extra mock parameters
-      metadata.parameters = Array.from({ length: 20 }, (_, i) => `Test: ${i}`);
-
+      metadata.inputs = Array.from({ length: 20 }, (_, i) => `Test: ${i}`);
       const wrapper = setup.mount(
-        <MetaData visible={true} metadata={metadata} />
+        <MetaDataList
```

Comment:
Given that this test focuses on testing the entire metadata panel and the integration of all its individual component, it would be better to follow the previous set up in mounting and test with the entire metadata component instead of the individual `MetaDataList` component - what is the reason behind changing the mount component?

Reply:
hi Susanna, please read the development notes I have mentioned it. Ideally the test should check if expand collapse function happens on metadata list if there's a limit and the new test does that. Currently, no metadata property uses this feature --parameters was using it previously but it no longer does.

Reply:
Hi Rashida, yes I have read the development notes you mentioned, and understand the situation where the metadata does not have a `parameter` field and is replaced b the `input` field. However, this does not affect the setup of this test - we are still importing `metadata` that includes the `inputs` field into the `metadata` component and the test should pick up the same CSS classes as we previously does. 

Setting up the test to only test the individual component essentially changes the nature of this test from an integration test to simply a unit test of the component.